conf/defaults.ini

```diff
@@ -1216,6 +1216,9 @@ enabled =
 # Comma-separated list of organization IDs for which to disable unified alerting. Only supported if unified alerting is enabled.
 disabled_orgs =
 
+# Specify how long to wait for the alerting service to initialize
+initialization_timeout = 60s
```

Comment:
Let's make the default 30s to keep the old value.

Reply:
Oops, good catch. Just changed this during my local testing :)